### ISSUE TITLE
AX5043 discontinuation

### ISSUE THREAD

**Issue by wimalopaan** [rdforrest/27MHz-using-AX5043#1]
Are you aware of any newer chips like the ax5043? 

I tries the same steps as you with this chip, find it very promising,  but realized that it is no longer available.

**Comment by rdforrest** [comment]
No, I'm not aware of any close alternative. I am now using Radiometrix modules which are much more expensive. 
I do know of a Dutch colleague who is continuing development (he has a large stock at home!) but I would need to contact him if you wish to be put in touch.

**Comment by wimalopaan** [comment]
Oh, yes, that would be very nice. I've also some chips in stock ;-)

**Comment by rdforrest** [comment]
I've passed your request on to him. If he agrees he will contact you
directly.

David

**Comment by wimalopaan** [comment]
Many thanks!

**Comment by rdforrest** [comment]
Rob says "yes" but please contact him via the contact form on his website:

www.robschuckman.nl

All the best for your project!

David

**Comment by wimalopaan** [comment]
Thanks.

I dropped a message on his project web site.

Interesing, that I found another article from him: https://www.electronicsworld.co.uk/building-and-evaluating-an-underwater-rf-communications-link/33705/

And he also describes a self-build 13MHz receiver! 
That's really cool, because I have designed (several) RC-transmitters (13-40MHz).

**Comment by rdforrest** [comment]
That's excellent. I have found Rob to be very helpful.
I think he is a professional RF engineer but with an amateur interest in
underwater communications. Hence my involvement. (I'm just an amateur radio
(Ham) & radio control submarine enthusiast.)
I have never got an AX5043 working properly but Rob has and apparently it
is a very good chip.
Good luck with the project!

David
M7DYP

**Comment by wimalopaan** [comment]
Mmh, I didn't get any confirmation from his web-site that my message got through. Is it normal that his web-site does not generate confirmation messages?

**Comment by rdforrest** [comment]
I would give it a day or two.
I can't remember if his site sends confirmation, but that is the way I
first contacted him.

David

**Comment by wimalopaan** [comment]
Thank you again for the contact to Rob. Unfortunately he wants to finish his project before going public with this.

Please excuse me for messaging you using this issue tracker... 

I made some more tests towards developing a SDR-based receiver (and transmitter) using a 2-FSK modulation. This has two modes of operation: compatibility to usual transmitters using FM-FSK (40MHz) but also can use a digital communication mode. The SDR receiver / transmitter can use all official frequencies (or 13MHz or 7MHz band if you want to). Actually I'm not sure if I should persue this project ... I saw that you also experimented with LoRa modules (468MHz). What is your opinion in this field? Would it be worthwhile in your opinion to further develop this SDR-receiver (the transmitter is completed).

Thnak you very much.
